### PR TITLE
[5] contact error when deleting alias

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #41713 .

### Summary of Changes
add check for hits

### Testing Instructions
Fill new contact under some category
Use "Save"
Use "Save as copy"
In this copy will be end of alias -2
Delete this end "-2" of alias
Save
Result: Save failed with the following error: Another Contact in this category has the same alias.
Delete all in alias line, to automaticaly generate new alias
Save
Result is empty alias(will not generate new one) and an ERROR: Save failed with the following error: Incorrect integer value: '

### Actual result BEFORE applying this Pull Request
Result is empty alias(will not generate new one) and an ERROR: Save failed with the following error: Incorrect integer value: '

### Expected result AFTER applying this Pull Request

no more error

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_contact/src/Table/ContactTable.php [hunk 1]
```diff
@@ -231,6 +231,10 @@ public function check()
             $this->modified_by = $this->created_by;
         }
 
+        if (empty($this->hits)) {
+            $this->hits =0;
```
Comment (richard67) [inline administrator/components/com_contact/src/Table/ContactTable.php]:
```suggestion
            $this->hits = 0;
```
Fix PHPCS failing due to missing space.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 873e44dbecb000d9e7e54ac51341cd2dbac03a0a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42968">issues.joomla.org/tracker/joomla-cms/42968</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42968">issues.joomla.org/tracker/joomla-cms/42968</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**